app/src/main/res/values/strings.xml

```diff
@@ -80,6 +80,7 @@
     <string name="main_deauthenticate">De-authenticate</string>
     <string name="main_clear_visitor_session">Clear Session</string>
     <string name="main_show_visitor_code">Show Visitor Code</string>
+    <string name="main_legacy_activity">Open legacy activity</string>
```

Comment:
Taking into account that it's an example app in our official public repo, it seems to me that we can consider a more informative text. For example, "Open non ComponentActivity" or "Open activity that doesn't support Results API".

Reply:
Comment in the PR header:

_Currently for testing purposes there is a button in MainFragment that opens a android.app.Activity type of view with what we can test this functionality. Before merger that button and view will be deleted._